Yoast/javascript
# Create TwitterSiteName component

## Summary

This PR can be summarized in the following changelog entry:

* Adds a TwitterSiteName component in SocialPreview\Twitter

## Relevant technical choices:

* Based on the branch of pull #56 (issue #44), seeming to have more commits than the case until that is merged.
* Does not do url to domain conversion in this component. Keeping it simple.
* Use TwitterPreview as base for now, to be able to easily add new components to later and to use it in the stand-alone.

## Test instructions

This PR can be tested by following these steps:

* Check the tests.
* Start the stand-alone.
* Go to the `TwitterPreview` page by clicking on the button in the top right.
* Check if it looks good. Mind you that the text container is not made, which will implement the sizes.

Requires #44 
Fixes #43
